docs/Основные блоки и устройство компьютера/Внутреннее устройство.md

```diff
@@ -0,0 +1,175 @@
+﻿# **Внутреннее устройство системного блока ПК**
+
+К внутренним устройствам относятся все комплектующие, которые располагаются и крепятся внутри компьютерного корпуса. Корпус ПК с установленными в нем комплектующими называется системным блоком (все компоненты внутри корпуса, объединены в рабочую систему).
+
+## **1. Материнская или системная плата («материнка» или «мать»)**
+
+Все компоненты компьютера связаны между собой одной самой большой печатной платой, её называют **системной платой** или **материнской платой** (в английском варианте **motherboard** или **mainboard**). Материнская плата первой устанавливается в корпус компьютера. На ней размещаются через распаянные разъемы или подключаются через специальные интерфейсные кабеля все внутренние устройства системного блока ПК и внешнее оборудование.
+
+Материнская плата является необходимым элементом для работоспособности компьютерной системы. Так же такими необходимыми элементами, которые устанавливаются или подключаются к ней являются:
+
+- Процессор с кулером.
+
+- Оперативная память.
+
+- Видеокарта (если нет встроенной в саму материнку или процессор).
+
+- Жесткий или SSD SATA диск (M.2 диск располагается на самой материнке).
+
+- Блок питания.
+
+- Корпус компьютера или что-то его заменяющее.
+
+Это необходимый минимум, чтобы ваш компьютер можно было запустить. Главная задача материнской платы-соединить все это оборудование и заставить его исправно работать.
+
+Основные характеристики:
+
+- Форм-фактор или размер. Подбирается под корпус или корпус под нее. Также от размера зависит функциональность и модернизируемость всего компа. Например, ATX материнки имеют 4 слота под оперативку, а Mini ITX уже только 2;
+
+- Процессорный сокет;
+
+- Чипсет.
+
+![enter image description here](http://it-uroki.ru/wp-content/uploads/2012/04/001-sistemnaya-plata.jpg)
```

Comment:
[ch-md-error-image-alt-text](https://chaberch.github.io/how-to-chaberch/markdown/#ch-md-error-image-alt-text)